src/main/java/net/wurstclient/mixin/BackgroundRendererMixin.java

```diff
@@ -29,4 +32,15 @@ private static void onGetFogModifier(Entity entity, float tickDelta,
 		if(WurstClient.INSTANCE.getHax().antiBlindHack.isEnabled())
 			ci.setReturnValue(null);
 	}
+	
+	@Redirect(at = @At(value = "INVOKE",
+		target = "Lnet/minecraft/entity/LivingEntity;hasStatusEffect(Lnet/minecraft/entity/effect/StatusEffect;)Z",
+		ordinal = 1), method = "render(Lnet/minecraft/client/render/Camera;FLnet/minecraft/client/world/ClientWorld;IF)V")
+	private static boolean onCheckDarkness(LivingEntity living, StatusEffect effect)
+	{
+		if(WurstClient.INSTANCE.getHax().antiBlindHack.isEnabled())
+			return false;
+		
+		return living.hasStatusEffect(effect);
+	}
```

Comment:
I'd prefer not to add a `@Redirect` here, since they always carry a risk of making Wurst incompatible with other mods. I have an idea that should get the same effect without one, and also happens to simplify AntiBlind's other mixins. Commit is coming up shortly.